# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/contentanalysis/tests/browser/browser_clipboard_content_analysis.js
# Commit: 366245552cae
# Full Hash: 366245552cae536ef883be806322395dc47b759d
# Author: Masayuki Nakano <masayuki@d-toybox.com>
# Date: 2025-06-17 09:24:05
# Regressor Bug: 1951038
# File Overlap Count: 1
# Description:
#   Bug 1951038 - Get rid of the legacy white-space normalizer of `HTMLEditor` and the pref r=m_kato
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D253589
# ==============================================================================

diff -r b13b0a530402 -r 366245552cae toolkit/components/contentanalysis/tests/browser/browser_clipboard_content_analysis.js
--- a/toolkit/components/contentanalysis/tests/browser/browser_clipboard_content_analysis.js	Tue Jun 17 00:10:27 2025 +0000
+++ b/toolkit/components/contentanalysis/tests/browser/browser_clipboard_content_analysis.js	Tue Jun 17 00:37:22 2025 +0000
@@ -37,9 +37,6 @@
   let tab = BrowserTestUtils.addTab(gBrowser);
   let browser = gBrowser.getBrowserForTab(tab);
 
-  const kNewWhiteSpaceNormalizerEnabled = Services.prefs.getBoolPref(
-    "editor.white_space_normalization.blink_compatible"
-  );
   gBrowser.selectedTab = tab;
 
   await promiseTabLoadEvent(tab, "data:text/html," + escape(testPage));
@@ -374,18 +371,15 @@
   // The new content should now include an image.
   await SpecialPowers.spawn(
     browser,
-    [allowPaste, plainTextOnly, kNewWhiteSpaceNormalizerEnabled],
-    (allowPaste, plainTextOnly, newWhiteSpaceNormalizerEnabled) => {
+    [allowPaste, plainTextOnly],
+    (allowPaste, plainTextOnly) => {
       var main = content.document.getElementById("main");
       let expectedContents;
       if (allowPaste) {
-        expectedContents = newWhiteSpaceNormalizerEnabled
-          ? '<i>Italic</i>&nbsp;<img id="img" tabindex="1" ' +
-            'src="http://example.org/browser/browser/base/content/test/general/moz.png">' +
-            "Test <b>Bold</b> After<b></b>"
-          : '<i>Italic</i> <img id="img" tabindex="1" ' +
-            'src="http://example.org/browser/browser/base/content/test/general/moz.png">' +
-            "Test <b>Bold</b> After<b></b>";
+        expectedContents =
+          '<i>Italic</i>&nbsp;<img id="img" tabindex="1" ' +
+          'src="http://example.org/browser/browser/base/content/test/general/moz.png">' +
+          "Test <b>Bold</b> After<b></b>";
       } else {
         // If plainTextOnly then no CA call will have been made, so
         // the content will be allowed. (but the earlier "<i>Italic</i>" part was not)